ftw/bob/web/template/+package.fullname+/+package.part_1+/+package.part_2+/profiles/default/workflows/+package.fullname_underscore+_workflow/specification.de.txt.bob

```diff
@@ -0,0 +1,75 @@
+[{{{package.fullname}}} workflow]
+
+Initialzustand: Entwurf
+
+Rollen Zuweisung:
+  Gast => Reader
+  Autor => Editor
+  Redaktor => Reviewer
+  Internetbenutzer => Anonymous
+  Verwalter => Site Administrator
```

Comment:
Ich würde die Rolle `Verwalter` umbenennen in `Webmaster`, das ist aus meiner Sicht die passendere Bezeichnung.